Packs/Tenable_io/Integrations/Tenable_io/Tenable_io_test.py

```diff
@@ -74,3 +183,59 @@ def test_get_vuln_by_asset(mocker, requests_mock):
 
     for k in actual_result[0].keys():
         assert EXPECTED_VULN_BY_ASSET_RESULTS[0][k] == actual_result[0][k]
+
+def test_get_scan_status_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scanId': '25'})
+    requests_mock.get(MOCK_PARAMS['url'] + 'scans/25', json={'info': {'status': 'canceled'}})
+
+    from Tenable_io import get_scan_status_command
+    results = get_scan_status_command()
+
+    entry_context = results['EntryContext']['TenableIO.Scan(val.Id === obj.Id)']
+
+    assert 'Scan status for 25' in results['HumanReadable']
+    assert entry_context['Id'] == '25'
+    assert entry_context['Status'] == 'canceled'
+
+def test_pause_scan_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scanId': '25'})
+    requests_mock.get(MOCK_PARAMS['url'] + 'scans/25', json={'info': {'status': 'canceled'}})
+    from Tenable_io import pause_scan_command
+    pause_scan_command()
+
+def test_resume_scan_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scanId': '25'})
+    requests_mock.get(MOCK_PARAMS['url'] + 'scans/25', json={'info': {'status': 'canceled'}})
+    from Tenable_io import resume_scan_command
+    resume_scan_command()
+
+def test_add_tags_command(mocker, requests_mock):
+    mock_demisto(mocker, {'payload': '25'})
+    requests_mock.post(MOCK_PARAMS['url'] + 'tags/values', json={'info': {'status': 'canceled'}})
+    from Tenable_io import add_tags
+    add_tags()
+
+def test_resume_scans_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scanIds': '25'})
+    requests_mock.get(MOCK_PARAMS['url'] + 'scans/25', json={'info': {'status': 'canceled'}})
+    from Tenable_io import resume_scans_command
+    resume_scans_command()
+
+def test_pause_scans_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scanIds': '25'})
+    requests_mock.get(MOCK_PARAMS['url'] + 'scans/25', json={'info': {'status': 'canceled'}})
+    from Tenable_io import pause_scans_command
+    pause_scans_command()
+
+def test_launch_scans_command(mocker, requests_mock):
+    mock_demisto(mocker, {'scan_ids': '25'})
+    requests_mock.post(MOCK_PARAMS['url'] + 'scans/25/launch', json={'info': {'status': 'canceled'}})
+    requests_mock.post(MOCK_PARAMS['url'] + 'scans/25/launch?alt_targets=10.0.0.1', json={'info': {'status': 'canceled'}})
+    from Tenable_io import launch_scans_command
+    launch_scans_command()
+
+def test_check_templates_command(mocker, requests_mock):
+    mock_demisto(mocker, {})
+    requests_mock.get(MOCK_PARAMS['url'] + 'editor/scan/templates', json={'info': {'status': 'canceled'}})
+    from Tenable_io import get_scan_templates
+    get_scan_templates()
```

Comment:
Can you explain the purpose of those tests?
Not sure why they don't have `asserts` in them.

Reply:
All of the commands here output nothing to use assert on.

Reply:
You can use `assert` to make sure the human-readable / context that the function returns are as they should be

Reply:
In-progress
Update 02.23.2021 1034 HRS EST:
Need to verify that there is a test function for each command

Reply:
Addressed
Update 02.23.2021 1034 HRS EST:
Each test is accounted for in the code, and the unneeded tests have been removed